vars.development-ssb.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Name prefix for applications in the logstack.
+app_name: logstack
+
+# Options to pass the JVM
+logstash_java_options: -Xms512m -Xmx512m
+# Number of instances of the Logstash to run.
+logstash_instances: 1
+# Amount of memory to provide to the Logstash (Java) application.
+logstash_memory: 1024
+# Assign route for the log shipper. This will be used as the logdrain for other
+# Cloud Foundry applications.
+hostname: logstash-development-datagov
+domain: app.cloud.gov
+https_proxy:
+
+# Optional license key for delivering logs to New Relic
+# See https://github.com/newrelic/logstash-output-plugin
+logstash_newrelic_license_key: d5cb2aa3c0e6523e95a70d50ad5fdbc4db22d9e7
```

Comment:
did we want to commit this to github? this key is compromised if so.

Reply:
yep, it is compromised. It is not a security risk but we do need to rotate it now.